docs/documentation/upgrading/topics/keycloak/changes-24_0_0.adoc

```diff
@@ -0,0 +1,4 @@
+= SPI Truststore Changes
+
+The spi-truststore-file-file is no longer required to initialize the spi provided truststore.  Instead the system truststore will be returned if the file is not set.
```

Comment:
Is `spi-truststore-file-file` and `spi-truststore-file-password` still supported? Will these be deprecated/removed at some point?

I don't really understand what's written here. I read this as I no longer need to use the mentioned SPI property to initialise the system truststore, or something like that, which makes no sense.

Reply:
The Truststore SPI might still make sense if someone wants a more granular control over truststores. Basically for use cases when setting system/global truststore is not desired. But now I'm not sure it's a real life use case. Do you suggest to deprecate the SPI config options?